internal/cmd/plugin/command_install.go

```diff
@@ -115,21 +126,21 @@ func installPlugin(manifest *Manifest, repositoryDir, installDir string) error {
 	return pluginInstaller.Install()
 }
 
-func getLanguage(manifest *Manifest) (string, *version.Version) {
+func getLanguage(manifest *Manifest) (string, *version.Version, error) {
 	if manifest == nil || len(manifest.Dependencies) == 0 {
-		return "", nil
+		return "", nil, errors.New("invalid plugin manifest: no language found")
```

Comment:
It feels weird to be using the colon in an error message without wrapping another error.
```suggestion
		return "", nil, errors.New("no language found in plugin manifest")
```